package.json

```diff
@@ -2,19 +2,28 @@
   "name": "atom-select-list",
   "version": "0.7.2",
   "description": "A general-purpose select list for use in Atom packages",
-  "main": "./src/select-list-view.js",
+  "main": "lib/select-list-view.js",
+  "files": [
+    "lib/**/*"
+  ],
   "scripts": {
```

Comment:
Hmm, might be worth adding a script to run `tsc watch`

Reply:
Fixed in https://github.com/atom/atom-select-list/pull/28/commits/31f366586af3d3de564f6c353e13e5130870f3d8